run.yaml

```diff
@@ -16,7 +16,7 @@ providers:
     - provider_id: trustyai_lmeval
       provider_type: remote::trustyai_lmeval
       config:
-        use_k8s: True
+        use_k8s: ${env.TRUSTYAI_LMEVAL_USE_K8S:True}
```

Comment:
I think this needs to be `${env.TRUSTYAI_LMEVAL_USE_K8S:=true}` based on https://llama-stack.readthedocs.io/en/latest/distributions/configuration.html#default-values?